.env.example

```diff
@@ -50,4 +50,18 @@ REACT_APP_GA_API_KEY=""
 # - used to fetch token balances on wallet
 # - you can pick up a Covalent key at https://www.covalenthq.com/
 REACT_APP_COVALENT=""
+
+# Zapper
+# Public key from Zapper, ONLY works on MAINNET
+# Zapper keys aren't needed unless you are trying to zap. 
+# Zaps aren't available except on mainnet
+REACT_APP_ZAPPER_API="96e0cc51-a62e-42ca-acee-910ea7d2a241"
+REACT_APP_ZAPPER_POOL="0x04f2694c8fcee23e8fd0dfea1d4f5bb8c352111f"
+
+# Olympus Give
+# These 2 settings should be toggled to "true" if you want to use Olympus Give features
+REACT_APP_GIVE_ENABLED="false"
```

Comment:
This isn't needed. It defaults to true, which it should.

https://github.com/OlympusDAO/olympus-frontend/blob/d657add2242553c3b18d5a836974fe2d043d6cc0/src/helpers/Environment.ts#L231

Reply:
Can you share the error messages? I can take a look and resolve them, if needed.

I would argue that Give should be enabled by default, though, and adding this line causes that to not be so.